src/cli2/merge.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import fs from "fs-extra";
+import stringify from "json-stable-stringify";
+import {join as pathJoin} from "path";
+
+import {LoggingTaskReporter} from "../util/taskReporter";
+import type {Command} from "./command";
+import {makePluginDir, loadInstanceConfig} from "./common";
+import {
+  toJSON as weightedGraphToJSON,
+  fromJSON as weightedGraphFromJSON,
+  type WeightedGraph,
+  merge,
+} from "../core/weightedGraph";
+
+function die(std, message) {
+  std.err("fatal: " + message);
+  return 1;
+}
+
+const mergeCommand: Command = async (args, std) => {
+  if (args.length !== 0) {
+    die(std, "usage: sourcecred merge");
```

Comment:
`return die`, no? lest we continue.

(The other commands miss this, too, but we can fix it here first.)

Reply:
Done.